### ISSUE TITLE
Typesense Facet_by Range not working as expected

### ISSUE THREAD

**Issue by kkarthik8224** [typesense/typesense#1389]
## Description

Hi, so I am actually trying to bucket the range for an integer field
price is configured as an integer field, with type as int64
## Steps to reproduce
executed this query
{
  "q": "ear rings",
  "query_by": "name",
  "facet_by": "price(a:[25000,50000])",
  "exhaustive_search": true,
  "page": 1,
  "per_page": 20,
  "group_by": "parent_id",
  "group_limit": "1"
}, f
## Expected Behavior
count should be more than zero, since my data set has products price ranging up to 500000, 

## Actual Behavior
  {
  "facet_counts": [
    {
      "counts": [
        {
          "count": 0,
          "highlighted": "",
          "value": "a"
        }
      ],
      "field_name": "price",
      "sampled": false,
      "stats": {
        "avg": 59798.57499686363,
        "max": 1253299,
        "min": 0,
        "sum": 4766544413,
        "total_values": 1
      }
    }
  

## Metadata

**Typesense Version**: 0.25.1

**Comment by aadarsh-nagrath** [comment]
It seems - count is coming up as zero even though your dataset has products with prices up to 500,000. My hunch is that the bucketing for the price range might be causing this.

Have you tried tweaking the facet_by parameter to ensure it covers the entire price range? Maybe something like "price(a:[0,500000])"?

**Comment by kishorenc** [comment]
@kkarthik8224 

Can you please post a minimally reproduceable example with a few sample records that I can run to see the problem in action?

**Comment by kkarthik8224** [comment]
@aadarsh-nagrath , Yes I have tried by tweaking the facet_by parameters for integer facets, range doesnt work if queried with group_by, the same works as expected if there is no group_by
@kishorenc

**Comment by kishorenc** [comment]
I think this is fixed in recent 0.25.2 RC builds. Can you please try against `0.25.2.rc10`?

**Comment by kkarthik8224** [comment]
Hi @kishorenc can i get the command to update to this 0.25.2 RC build for deb and linux
